### PR TITLE
WEBAPP-130: Disable service-worker caching

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -100,11 +100,6 @@ ReactDOM.render(App, container)
 // Sets the splash to hidden when the page is rendered
 document.getElementById('splash').className += ' splash-hidden'
 
-if (navigator.serviceWorker) {
-  navigator.serviceWorker.register('/sw.js', {scope: '/'})
-    .catch((err) => console.error('Unable to register service worker.', err))
-}
```
Comment (Schedulaar) [inline src/main.js]: Maybe we need to unregister the current ServiceWorker... Otherwise the service worker may still be in place. 🤔

Comment (maxammann) [reply src/main.js]: I think sw.js is no longer available to it should fail to start

Comment (Schedulaar) [reply src/main.js]: I think sw.js is probably cached by the service worker itself :D

Comment (Schedulaar) [reply src/main.js]: https://stackoverflow.com/questions/33704791/how-do-i-uninstall-a-service-worker

Comment (maxammann) [reply src/main.js]: lool you could be right 😭 
k Ill add code to unregister all

Comment (maxammann) [reply src/main.js]: done

**Pull request closed without merging**